dependencies.yaml

```diff
@@ -175,6 +185,15 @@ dependencies:
               cuda: "11.2"
             packages:
               - cudatoolkit=11.2
+          - matrix:
+              cuda: "12.0"
```

Comment:
Put this list at the top so it is sorted from 12.0 to 11.2. Then drop all but the -dev packages. Don’t specify their versions, just pin cuda-version=12.0 in that list.